components/TextFields/examples/experimental/supplemental/MDCBaseInputChipView+MaterialTheming.m

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBaseInputChipView+MaterialTheming.h"
+
+#import <Foundation/Foundation.h>
+
+#import "MDCBaseInputChipView+MDCContainedInputView.h"
+#import "MDCContainedInputView.h"
+
+@implementation MDCBaseInputChipView (MaterialTheming)
+
+- (void)applyThemeWithScheme:(nonnull id<MDCContainerScheming>)containerScheme {
+  [self applyTypographyScheme:[self typographySchemeWithContainerScheme:containerScheme]];
+  [self applyDefaultColorTheme:[self colorSchemeWithContainerScheme:containerScheme]];
+}
+
+- (void)applyErrorThemeWithScheme:(nonnull id<MDCContainerScheming>)containerScheme {
+  [self applyTypographyScheme:[self typographySchemeWithContainerScheme:containerScheme]];
+  [self applyErrorColorTheme:[self colorSchemeWithContainerScheme:containerScheme]];
+}
+
+- (id<MDCColorScheming>)colorSchemeWithContainerScheme:
+    (nonnull id<MDCContainerScheming>)containerScheme {
+  id<MDCColorScheming> mdcColorScheme = containerScheme.colorScheme;
+  if (!mdcColorScheme) {
```

Comment:
This conditional code is no longer needed and probably incorrect after we changed color schemes to be nonnull. I believe you can safely assume the container scheme has a color/typography scheme.
(here any everywhere throughout the code).